### PR TITLE
Correct tests for productized artifacts

### PR DESCRIPTION
**Pull request by radtriste**
- Test updates are done because we don't have process artifacts in productized version ... but no backport of those modifications because process will be included in the next version, so no need.
- Update on test scripts will be backported to master once this one is approved and merged

### CODE REVIEW

File: tests/features/kogito-common-builder-jvm.feature [hunk 1]
```diff
@@ -77,16 +77,6 @@ Feature: kogito-builder image JVM build tests
       | expected_phrase | ["hello","world"]     |
     And file /home/kogito/bin/quarkus-run.jar should exist
 
-  Scenario: Verify if the s2i build is finished as expected performing a non native build with persistence enabled
```
Comment (radtriste) [inline tests/features/kogito-common-builder-jvm.feature:80]: Moved to `kogito-builder.feature`

File: tests/features/kogito-common-builder-jvm.feature [hunk 2]
```diff
@@ -181,106 +171,97 @@ Feature: kogito-builder image JVM build tests
...
       | request_method       | POST                                                                          |
-      | request_body         | {"approver" : "john", "order" : {"orderNumber" : "12345", "shipped" : false}} |
+      | request_body         | {"maxAmount":5000,"loanApplications":[{"id":"ABC10001","amount":2000,"deposit":100,"applicant":{"age":45,"name":"John"}}]} |
       | content_type         | application/json                                                              |
       | expected_status_code | 201                                                                           |
-    And file /home/kogito/bin/process-springboot-example.jar should exist
+    And file /home/kogito/bin/ruleunit-springboot-example.jar should exist
     And container log should contain Tomcat initialized with port(s): 8080 (http)
-
-  Scenario: Verify if the s2i build is finished as expected with persistence enabled
```
Comment (radtriste) [inline tests/features/kogito-common-builder-jvm.feature:236]: Moved to `kogito-builder.feature`

File: scripts/update-tests.py [hunk 3]
```diff
@@ -18,6 +18,8 @@
     parser.add_argument('--examples-ref', dest='examples_ref', help='To update the examples ref for testing')
     parser.add_argument('--artifacts-version', dest='artifacts_version', help='To update the artifacts version for testing')
     parser.add_argument('--ignore-self-signed-cert', dest='ignore_self_signed_cert', default=False, action='store_true', help='If set to true will relax the SSL for user-generated self-signed certificates')
+    parser.add_argument('--runtime-image-jvm', dest='runtime_image_jvm', help='To update the runtime jvm image for testing')
+    parser.add_argument('--runtime-image-native', dest='runtime_image_native', help='To update the runtime native image for testing')
```
Comment (sutaakar) [inline scripts/update-tests.py]: What about builder image? I would expect that it needs to be updated too.

Comment (radtriste) [reply scripts/update-tests.py]: in fact, we need to update that image in the steps of behave tests as it is used directly ... we don't need to update the builder in them.

Comment (sutaakar) [reply scripts/update-tests.py]: ok

**Review by spolti — APPROVED** [review]
looks good to me.

**Review by sutaakar — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
will these changes be needed for 1.6.x and master branches?

**Comment by radtriste** [comment]
I will backport the needed changes on Monday

**Pull request merged**